### ISSUE TITLE
Bug: Legend label name replacement passes through when labels aren't found

### ISSUE THREAD

**Issue by brokenjacobs** [VictoriaMetrics/grafana-datasource#133]
In grafana, if you specify a label name using the `{{ label }}` syntax that is missing in the result, when using this datasource plugin, the missing label name will pass through into the result. For instance:
```
max by (alertname) (
  max_over_time(ALERTS{alertname="Watchdog"}[$__interval])
  or
  max_over_time(ALERTS{severity=~"ok|OK|Ok"}[$__interval]) * 2
)
```

When using a label legend of:
{{alertname}} {{monkey}}

It renders as if monkey was a label in the result set:
![image](https://github.com/VictoriaMetrics/grafana-datasource/assets/3159463/d256a206-7ee0-4f05-99b6-ffefdf26988b)

When using the prometheus datasource missing labels do not render at all. I don't know where in the provider this behavior occurs but we've noticed it in multiple dashboards when trying to use it.

**Comment by hagen1778** [comment]
Hi @brokenjacobs!
The issue was addressed in https://github.com/VictoriaMetrics/grafana-datasource/issues/133 and will be available in the next release. I'll keep this issue open until the next release is cut.

**Comment by Loori-R** [comment]
This issue has been fixed in datasource starting from [v0.6.0](https://github.com/VictoriaMetrics/grafana-datasource/releases/tag/v0.6.0). Closing the issue as fixed.
